### ISSUE TITLE
added separator for VERTICAL ActionItem 

### ISSUE THREAD

**Issue by milaptank** [piruin/quickaction#11]
added separator for VERTICAL actionItem also 
change this code at this [LINE](https://github.com/piruin/quickaction/blob/master/quickaction/src/main/java/me/piruin/quickaction/QuickAction.java#L214)
```java
  private void addActionView(int position, View actionView) {
        if (orientation == HORIZONTAL && position != 0) {
            position *= 2;
            int separatorPos = position - 1;
            View separator = new View(getContext());
            separator.setBackgroundColor(Color.argb(32, 0, 0, 0));
            int width = resource.getDimensionPixelOffset(R.dimen.separator_width);
            track.addView(separator, separatorPos, new LayoutParams(width, MATCH_PARENT));
        } else if (orientation == VERTICAL && position != 0) {
            position *= 2;
            int separatorPos = position - 1;
            View separator = new View(getContext());
            separator.setBackgroundColor(Color.argb(32, 0, 0, 0));
            int width = resource.getDimensionPixelOffset(R.dimen.separator_width);
            track.addView(separator, separatorPos, new LayoutParams(MATCH_PARENT, width));
        }
        track.addView(actionView, position);
    }
```

**Comment by piruin** [comment]
By design, QuickAction shouldn't need separator when use vertical orientation. but if use really need it, Can you add option for library user to turn on-off separator (default should be same old behavior)?

**Comment by milaptank** [comment]
okay yes we can give option to user  turn on-off separator  both way VERTICAL as well HORIZONTAL 
but  default should be old behavior

**Comment by piruin** [comment]
Good, I will waiting for your new pull request

**Comment by piruin** [comment]
Released at version `2.4`
